pkgs/development/tools/out-of-tree/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildGoPackage, fetchgit, qemu, docker }:
+
+buildGoPackage rec {
+  name = "out-of-tree-${version}";
+  version = "1.0.1";
+  rev = "v${version}";
+
+  buildInputs = [ qemu docker ];
+
+  goPackagePath = "code.dumpstack.io/tools/out-of-tree";
+
+  src = fetchgit {
+    inherit rev;
+    url = "https://code.dumpstack.io/tools/out-of-tree.git";
+    sha256 = "0p0ps73w6lmsdyf7irqgbhfxjg5smgbn081d06pnr1zmxvw8dryx";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    description = "out-of-tree kernel {module, exploit} development tool";
```

Comment:
We have a rule that the `description` shouldn't have the actual package name restated in it at the beginning.

Reply:
fixed